arbor/include/arbor/spike_source_cell.hpp

```diff
@@ -11,10 +11,12 @@ namespace arb {
 
 struct ARB_SYMBOL_VISIBLE spike_source_cell {
     cell_tag_type source; // Label of source.
-    schedule seq;
+    std::vector<schedule> seqs;
 
     spike_source_cell() = delete;
-    spike_source_cell(cell_tag_type source, schedule seq): source(std::move(source)), seq(std::move(seq)) {};
+    template<typename... Seqs>
+    spike_source_cell(cell_tag_type source, Seqs... seqs): source(std::move(source)), seqs{seqs...} {};
```

Comment:
As written `seqs` is passed by value, thus copied into the argument list and then moved into the class
(actually copied again, but let's trust in the compiler for a sec).

Since we are in C++17 this copy is elided, see various notes on the topic, here's one I
could summon quickly https://andreasfertig.blog/2022/02/why-you-should-use-stdmove-only-rarely/.

Quoting the relevant bit

> Due to the mandatory copy elision in C++17, even with <code>-fno-elide-constructors</code>, the compiler must now elide this nonsense construction. <em>However</em>, if we apply <code>std::move</code> to the temporary copy elision doesn't apply anymore, and we're back in seeing a move construction. 

Am I missing something?